scripts/ci/qemu-run.sh

```diff
@@ -1,27 +1,57 @@
 #!/usr/bin/env bash
 
-if [ ! -f "./OVMF_CODE.fd" ]; then
-    wget -O ./OVMF_CODE.fd https://github.com/Dasharo/edk2/releases/download/dasharo_qemu_v0.0.1-test3/OVMF_CODE_RELEASE.fd
+INIT_DIR="$(pwd)"
+
+# Check if a directory was provided as the first parameter
+if [ -n "$1" ]; then
+    DIR="$1"
+else
+    DIR="$(pwd)"
 fi
 
-if [ ! -f "./OVMF_VARS.fd" ]; then
+# Check if the directory exists
+if [ ! -d "$DIR" ]; then
+    echo "The provided directory does not exist."
+    exit 1
+fi
+
+# Change to the specified directory
+cd "$DIR" || exit
+
+# Check for the existence of OVMF_CODE.fd and OVMF_VARS.fs files
+if [ ! -f "OVMF_CODE.fd" ] || [ ! -f "OVMF_VARS.fd" ]; then
+    echo "The required files OVMF_CODE.fd and OVMF_VARS.fs are missing."
+    echo "Downloading files from the server..."
+    wget -O ./OVMF_CODE.fd https://github.com/Dasharo/edk2/releases/download/dasharo_qemu_v0.0.1-test3/OVMF_CODE_RELEASE.fd
     wget -O ./OVMF_VARS.fd https://github.com/Dasharo/edk2/releases/download/dasharo_qemu_v0.0.1-test3/OVMF_VARS_RELEASE.fd
+else
+    echo "Both OVMF_CODE.fd and OVMF_VARS.fs files exist in the directory."
 fi
 
+echo "Copy OVMF_VARS.fd to /tmp/OVMF_VARS.fd"
 cp ./OVMF_VARS.fd /tmp/OVMF_VARS.fd
 
 q35_params="-machine q35,smm=on \
+    -m 1G \
     -global driver=cfi.pflash01,property=secure,value=on \
     -drive if=pflash,format=raw,unit=0,file=./OVMF_CODE.fd,readonly=on \
     -drive if=pflash,format=raw,unit=1,file=/tmp/OVMF_VARS.fd \
     -debugcon file:debug.log -global isa-debugcon.iobase=0x402 \
     -global ICH9-LPC.disable_s3=1 \
     -qmp unix:/tmp/qmp-socket,server,nowait \
-    -net none \
```

Comment:
@pietrushnic Is some additional setup required on the host for networking?

I quickly tried to run your code, but cannot enter iPXE setup

![image](https://github.com/Dasharo/open-source-firmware-validation/assets/19353650/97793f23-8239-4048-b194-be63075886a8)

Reply:
Sorry, I have in fact used incorrect revision of the edk2. I have downloaded artifacts from: https://github.com/Dasharo/edk2/actions/runs/6632586157 and can see iPXE menu.